dashboards/compute/compute_instance_dashboard.sp

```diff
@@ -0,0 +1,287 @@
+dashboard "ibm_compute_instance_dashboard" {
+
+  title         = "IBM Compute Instance Dashboard"
+  documentation = file("./dashboards/compute/docs/compute_instance_dashboard.md")
+
+  tags = merge(local.compute_common_tags, {
+    type = "Dashboard"
+  })
+
+  container {
+
+    # Analysis
+    card {
+      sql   = query.ibm_compute_instance_count.sql
+      width = 2
+    }
+
+    card {
+      sql   = query.ibm_compute_instance_total_vcpu.sql
+      width = 2
+    }
+
+    # Assessments
+    card {
+      sql   = query.ibm_compute_public_instance_count.sql
+      width = 2
+    }
+
+  }
+
+  container {
+
+    title = "Assessments"
+    width = 12
+
+    chart {
+      title = "Public/Private"
+      sql   = query.ibm_compute_instance_by_floating_ip.sql
+      type  = "donut"
+      width = 3
+
+      series "count" {
+        point "private" {
+          color = "ok"
+        }
+        point "public" {
+          color = "alert"
+        }
+      }
+    }
+
+  }
+
+  container {
+
+    title = "Analysis"
+
+    chart {
+      title = "Instances by Account"
+      sql   = query.ibm_compute_instance_by_account.sql
+      type  = "column"
+      width = 3
+    }
+
+    chart {
+      title = "Instances by Resource Group"
+      sql   = query.ibm_compute_instance_by_resource_group.sql
+      type  = "column"
+      width = 3
+    }
+
+    chart {
+      title = "Instances by Region"
+      sql   = query.ibm_compute_instance_by_region.sql
+      type  = "column"
+      width = 3
+    }
+
+
+    chart {
+      title = "Instances by Zone"
+      sql   = query.ibm_compute_instance_by_zone.sql
+      type  = "column"
+      width = 3
+    }
+
+    chart {
+      title = "Instances by Status"
+      sql   = query.ibm_compute_instance_by_status.sql
+      type  = "column"
+      width = 3
+    }
+
+    chart {
+      title = "Instances by Age"
+      sql   = query.ibm_compute_instance_by_creation_month.sql
+      type  = "column"
+      width = 3
+    }
+
+    chart {
+      title = "Instances by Architecture"
+      sql   = query.ibm_compute_instance_by_architecture.sql
+      type  = "column"
+      width = 3
+    }
+
+  }
+
+}
+
+# Card Queries
+
+query "ibm_compute_instance_count" {
+  sql = <<-EOQ
+    select count(*) as "Instances" from ibm_is_instance;
+  EOQ
+}
+
+query "ibm_compute_instance_total_vcpu" {
+  sql = <<-EOQ
+    select
+      sum((vcpu ->> 'count')::int) as "Total vCPUs"
```

Comment:
```suggestion
      sum((vcpu ->> 'count')::int) as "vCPUs"
```
Total mostly we use in dashboard while consolidating, here we can show only `vCPUs`